melusine/models/train.py

```diff
@@ -578,5 +621,6 @@ def _get_meta(self, X):
                 X_meta = X[meta_columns_list]
 
             nb_meta_features = len(meta_columns_list)
-
+        if isinstance(X_meta, pd.DataFrame):
```

Comment:
we could avoir this import of pandas by replacing is instance by str(type(X_meta)) == 'pd.DataFrame' but I'm not sure